ladybug/hourlyplot.py

```diff
@@ -0,0 +1,424 @@
+# coding=utf-8
+from __future__ import division
+
+from .analysisperiod import AnalysisPeriod
+from .datacollection import HourlyContinuousCollection, HourlyDiscontinuousCollection
+from .graphic import GraphicContainer
+from .legend import LegendParameters
+
+from ladybug_geometry.geometry2d.pointvector import Point2D, Vector2D
+from ladybug_geometry.geometry2d.line import LineSegment2D
+from ladybug_geometry.geometry2d.polygon import Polygon2D
+from ladybug_geometry.geometry2d.polyline import Polyline2D
+from ladybug_geometry.geometry2d.mesh import Mesh2D
+from ladybug_geometry.geometry3d.pointvector import Point3D, Vector3D
+from ladybug_geometry.geometry3d.line import LineSegment3D
+from ladybug_geometry.geometry3d.plane import Plane
+from ladybug_geometry.geometry3d.polyline import Polyline3D
+from ladybug_geometry.geometry3d.mesh import Mesh3D
```

Comment:
Not related to this PR but just seeing this it might be a good idea to add the commonly used objects to `__init__` file to make it easier to access them for a user.

```python
from ladybug_geometry.geometry2d import Point2D, Vector2D
```

Reply:
Ok. I opened an issue
https://github.com/ladybug-tools/ladybug-geometry/issues/100